source/slang/slang-parser.cpp

```diff
@@ -6296,6 +6302,7 @@ namespace Slang
         _makeParseExpr("This",  parseThisTypeExpr),
         _makeParseExpr("true",  parseTrueExpr),
         _makeParseExpr("false", parseFalseExpr),
+        _makeParseExpr("nullptr", parseNullPtrExpr),
```

Comment:
I understand why you use `nullptr` here, but I would personally greatly prefer `null`. I suspect that we will end up needing `nullptr` as well, even if we add `null`, but just copy-pasting the C++ name for things like this always worries me.

Reply:
I reserved `null` for `ref` classes, and `nullptr` is just for pointers. This is likely unnecessary, but we can change it if we want.